drivers/SmartThings/matter-sensor/profiles/motion-battery.yml

```diff
@@ -1,4 +1,4 @@
-name: matter-motion-battery
+name: motion-battery
```

Comment:
I know that deleting profiles can cause some trouble, but is renaming them okay or would this lead to similar issues? I renamed this so that it would follow the naming scheme used to construct the profile name in `do_configure`

Reply:
I believe it leaves users who have already installed the device with this profile in an "orphaned" state just the same as a delete.

Reply:
I dont think this actually causes problems though. Those devices should still function.